cpu/stm32l0/include/periph_cpu.h

```diff
@@ -85,6 +85,17 @@ typedef struct {
     uint8_t ev_irqn;        /**< event IRQ */
 } i2c_conf_t;
 
+/**
+ * @brief   Define the number of available PM modes
+ */
+#define PM_NUM_MODES    (2U)
+
+/**
+ * @brief   Override the default initial PM blocker
+ * @todo   we block all modes per default, until PM is cleanly implemented
```

Comment:
What is missing with PM to be 'cleanly implemented' ?

Reply:
> Do you thing it is possible to associate xtimer with low power timer (LPTIM1) from stm32l0 so that we still have a time base running in stop mode ?

I don't know. Maybe @haukepetersen or @kaspar030 have an idea since they are working on an improved version of the xtimer that is able to work in low power modes (IIRC) ?